resources/views/app.blade.php

```diff
@@ -8,24 +8,28 @@
 
     <title inertia>{{ config('app.name', 'Laravel') }}</title>
 
+    <link rel="shortcut icon" href="img/favicon.svg">
+
     <!-- Scripts -->
     @if (app()->bound('sentry') && config('sentry.dsn') !== null)
     <script type="text/javascript">
-      const SentryConfig = {!! \json_encode([
-        'dsn' => config('sentry.dsn'),
-        'environment' => config('sentry.environment'),
-        'sendDefaultPii' => config('sentry.send_default_pii'),
-        'tracesSampleRate' => config('sentry.traces_sample_rate'),
-      ]); !!}
+      const SentryConfig = {
+        !!\json_encode([
+          'dsn' => config('sentry.dsn'),
+          'environment' => config('sentry.environment'),
+          'sendDefaultPii' => config('sentry.send_default_pii'),
+          'tracesSampleRate' => config('sentry.traces_sample_rate'),
+        ]);!!
+      }
```

Comment:
This is not good, Laravel expect that:
```suggestion
      const SentryConfig = {!! \json_encode([
        'dsn' => config('sentry.dsn'),
        'environment' => config('sentry.environment'),
        'sendDefaultPii' => config('sentry.send_default_pii'),
        'tracesSampleRate' => config('sentry.traces_sample_rate'),
      ]); !!}
```

Reply:
Well, VSCode somehow has changed that... I didn't touch it myself.